nlp-services/asr/service/ASRService.py

```diff
@@ -0,0 +1,26 @@
+import wave
+
+import deepspeech
+import numpy as np
+
+model_en = deepspeech.Model('models/deepspeech-0.9.3-models.pbmm')
+model_en.enableExternalScorer('models/deepspeech-0.9.3-models.scorer')
+
+model_de = deepspeech.Model('models/model_de.pbmm')
+model_de.enableExternalScorer('models/scorer_de.scorer')
+
+class ASRService():
+    active_model = None
+
```

Comment:
Maybe it would be enough here to maintain a single function of _speech_to_text_ which decides which model to use based on a _lang_ parameter. This way we could delete the lang specific functions as they only set the active model and we also don't really plan additional functions in the ASRService if I'm right. 

Maybe the metadata handling would be present here.

Reply:
Thank you for the review! I am going to refactor the code accordingly!